backend/src/main/java/com/bakdata/conquery/commands/WorkerAPI.java

```diff
@@ -0,0 +1,83 @@
+package com.bakdata.conquery.commands;
+
+import java.util.ArrayList;
+import java.util.stream.Collectors;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+
+import com.bakdata.conquery.io.jersey.ExtraMimeTypes;
+import com.bakdata.conquery.models.events.Bucket;
+import com.bakdata.conquery.models.identifiable.ids.specific.DatasetId;
+import com.bakdata.conquery.models.worker.Workers;
+import com.bakdata.conquery.resources.ResourceConstants;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Consumes({ExtraMimeTypes.JSON_STRING, ExtraMimeTypes.SMILE_STRING})
+@Produces({ExtraMimeTypes.JSON_STRING})
+@Path("workers")
+@RequiredArgsConstructor
+@Getter
+public class WorkerAPI {
+
+	private final Workers workers;
```

Comment:
Jersey erwartet in einer Resource glaube ich nur Methoden die Endpunkte sind:

https://dev.azure.com/bakdata/public/_build/results?buildId=4423&view=logs&j=51d5ae3f-2fe7-559b-03ed-ca133007d3ee&t=0036330d-ce57-5434-6c41-012e08b63e82&l=2641